src/credentials/providers/credentialsProviderId.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const CREDENTIALS_PROVIDER_ID_SEPARATOR = ':'
```

Comment:
See discussion this morning, we should probably add another field instead, or change to '|'

Reply:
Changed to |